src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,101 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+import '@fortawesome/fontawesome-free/css/all.css';
+import 'bulma/css/bulma.css';
+
+export const PeoplePage = () => {
+  const { personId } = useParams();
```

Comment:
```suggestion
  const { slug } = useParams();
```

You don't have personId in your router